src/sources/aws_kinesis_firehose/handlers.rs

```diff
@@ -41,12 +42,16 @@ pub async fn firehose(
                 request_id: request_id.clone(),
             })
             .map_err(reject::custom)?;
+        emit!(&BytesReceived {
+            byte_size: bytes.len(),
+            protocol: "http",
+        });
 
         let mut stream = FramedRead::new(bytes.as_ref(), decoder.clone());
         loop {
             match stream.next().await {
                 Some(Ok((events, byte_size))) => {
-                    emit!(&AwsKinesisFirehoseEventsReceived {
+                    emit!(&EventsReceived {
                         count: events.len(),
                         byte_size
```

Comment:
Same here, this `byte_size` should be using `ByteSizeOf::size_of` rather than the raw bytes.